Formula/eventql.rb

```diff
@@ -0,0 +1,25 @@
+class Eventql < Formula
+  desc "Database for large-scale event analytics"
+  homepage "https://eventql.io"
+  url "http://dl.eventql.io/eventql/v0.3.2/eventql-0.3.2.tgz"
+  sha256 "d235f3e78fa5f6569fc2db94161d3e3f9cb71dc0646e341acd91814cefd23640"
+  head "https://github.com/eventql/eventql.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    ENV.deparallelize
+    system "./autogen.sh" if build.head?
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/evql", "--help"
```

Comment:
```
system bin/"evql", "--help"
```

But is it possible to write a more substantial test that actually tries to do something with `evql`?